src/Joonasw.AspNetCore.SecurityHeaders/Csp/Builder/CspBuilder.cs

```diff
@@ -81,6 +81,11 @@ public class CspBuilder
         /// Setups up rules for controlling when subresource integrity must be used
         /// </summary>
         public CspRequireSriBuilder RequireSri { get; } = new CspRequireSriBuilder();
+        
+        /// <summary>
+        /// Setups up rules for controlling when to restrict the creation of Trusted Types policies
```

Comment:
Typo: "Setups up" -> "Sets up". I do notice this typo is in existing code as well :\

It could be good to link to the MDN documentation here.